apps/web-portal-client/lib/components/HomePage.jsx

```diff
@@ -0,0 +1,73 @@
+import React, {useState} from 'react'
+
+import {Logo} from './Logo'
+import {Card} from './Card'
+import {UserProfile} from './UserProfile'
+import {LoginButton} from './LoginButton'
+
+import {DesktopMenu, MobileMenu, MobileMenuButton} from './main-menu'
+
+export function HomePage() {
+  const [menuStatus, setMenuStatus] = useState(false)
+  const [loginStatus, setLoginStatus] = useState(false)
+
+  return (
+    <div>
+      <nav className="bg-gray-800">
+        <div className="max-w-7xl mx-auto px-4 sm:px-6 lg:px-8">
+          <div className="flex items-center justify-between h-16">
+            <div className="flex items-center">
+              <Logo />
+              <DesktopMenu
+                items={[
+                  {desc: 'Home', link: '#', active: true},
+                  {desc: 'Product', link: '#'},
+                  {desc: 'Teams', link: '#'},
+                  {desc: 'About', link: '#'},
+                ]}
+              />
+            </div>
+            <div className="flex space-x-1">
+              <LoginButton loginStatus={loginStatus} setLoginStatus={setLoginStatus} />
+              <MobileMenuButton menuStatus={menuStatus} setMenuStatus={setMenuStatus} />
+            </div>
+          </div>
+        </div>
+        <MobileMenu
```

Comment:
it is not clear usage of two components MobileMenu and DesctopMenu in one component together with the other components.
Try to join these two components in one with the name Menu and implement the logic that switches between the menu types.

Reply:
I've thought about it. But these component are different. Different markup, classes and so on. Even items that we pass to these components can be different. That's why I decided to make two differenr components

Reply:
Looks like we already have a common component for them, it's a `NavBar`